stable/awsebscsiprovisioner/Chart.yaml

```diff
@@ -5,7 +5,8 @@ name: awsebscsiprovisioner
 maintainers:
   - name: alejandroEsc
   - name: gpaul
-version: 0.2.8
```

Comment:
Please rebase on version 0.3.0.

Reply:
that is not truth https://github.com/mesosphere/charts/blob/dev/stable/awsebscsiprovisioner/Chart.yaml#L8 :)

Reply:
You're right, I looked at the wrong chart :)